nixos/modules/services/misc/gitlab.nix

```diff
@@ -613,7 +877,8 @@ in {
     };
 
     systemd.services.gitlab = {
-      after = [ "gitlab-workhorse.service" "gitaly.service" "network.target" "postgresql.service" "redis.service" ];
+      after = [ "gitlab-workhorse.service" "gitaly.service" "network.target" "postgresql.service" "redis.service" "keys.target" ];
```

Comment:
If there were a `services.gitlab.database.createLocally` option you could make `"postgresql.service"` optional here as otherwise you wouldn't have a dependency on a service which is likely not even running.

Reply:
Yeah, sounds like a good idea. :)

Reply:
@aanderse `after` doesn't mean it depends on it, see `systemd.unit(5)`

Reply:
@lheckemann I understand but had a poor choice of wording in this case. Sorry. When I said "depends" I didn't mean it in the `systemd` sense of the word, I just meant it was listed, when though not being used. Always nice to have clear and concise code.